### DISCUSSION TITLE
Bugs? SOC is not zero after discharging.

### DISCUSSION THREAD

**Discussion by L4XIA** [pybamm-team/PyBaMM#2557] [Q&A] [answered]
Slightly modified the code, I found that the SOC cannot go back to zero after discharging (see the figure below). Is this a bug?
<img width="518" alt="SOC is not zero after discharging" src="https://user-images.githubusercontent.com/4371362/207633590-d49d4386-8a1a-490b-80f0-dd841a3ae54a.PNG">

below is the related code

import pybamm as pb
import matplotlib.pyplot as plt
pb.set_logging_level("NOTICE")
model = pb.lithium_ion.DFN(
    {
        "SEI": "ec reaction limited",
        "SEI film resistance": "distributed",
        "SEI porosity change": "true",
        "lithium plating": "irreversible",
        "lithium plating porosity change": "true",
    }
)

param = pb.ParameterValues("Mohtat2020")
experiment = pb.Experiment(
    [
        (
            "Charge at 1 C until 4.2 V",
            "Hold at 4.2 V until C/10",
            "Rest for 5 minutes",
            "Discharge at 1 C until 2.8 V",
            "Rest for 5 minutes",
        )
    ]
    * 2,
    period = "5 seconds",

)

sim = pb.Simulation(model, experiment=experiment, parameter_values=param)
sim.solve(solver=pb.CasadiSolver(mode="safe"))
sol         = sim.solution
t      = sol["Time [min]"].entries
vv     = sol["Terminal voltage [V]"].entries
SOC    = sol["Negative electrode SOC"].entries*100

fig, axs = plt.subplots(figsize=(13,9))
axs.plot(SOC, vv, color="blue", linestyle="solid", label="Simulation")

axs.set_xlabel("SOC [%]")
  
axs.legend()
axs.set_ylabel("Voltage [V]")
axs.grid(True)

**Comment by valentinsulzer** [comment, marked as answer]
- "Negative electrode SOC" is not the same as cell SOC (we just renamed it to "Negative electrode stoichiometry" to make this clearer)
- It would probably go back to zero if you included a CV hold at 2.8V
- Cell SOC is not currently available but should be soon, see #2553 for a discussion

**Reply by L4XIA** [reply]
Thanks for clarifications!
